### DISCUSSION TITLE
Security scan is queued

### DISCUSSION THREAD

**Discussion by JagadeeshRam** [quay/clair#1356] [Q&A]
I have setup the clair in locally as mentioned in the https://quay.github.io/clair/howto/testing.html . after pushing image to quay it was showing "Queued" in the security scan
![Screenshot from 2021-09-03 21-10-06](https://user-images.githubusercontent.com/55401565/132032005-234d9743-c458-4ef8-a097-88f67a3de292.png)

Please give me solution for this

**Comment by hdonnay** [comment]
You've provided no information to diagnose a problem.
